src/Pokemon.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Pokedex;
+
+class Pokemon
+{
+
+    private $id;
+    private $name;
+    private $type_1;
+    private $type_2;
+
+
+    public function __construct()
+    {
+        //capitalises first letter of each string for more a professional appearance
```

Comment:
Is this comment necessary?  This should probably be in the docblock.

Reply:
Solved in b605239

Reply:
Solved in b605239